VideoLocker/res/layout/fragment_my_all_videos.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Interesting; I didn't know about the existence of the `layout` tag before. What is the purpose of it?